graylog2-web-interface/src/stores/lookup-tables/LookupTableDataAdaptersStore.js

```diff
@@ -118,7 +118,7 @@ const LookupTableDataAdaptersStore = Reflux.createStore({
   },
 
   lookup(adapterName, key) {
-    const promise = fetch('GET', this._url(`adapters/${adapterName}/query?key=${key}`));
+    const promise = fetch('GET', this._url(`adapters/${adapterName}/query?key=${encodeURIComponent(key)}`));
```

Comment:
Can you please apply the same change to the lookup function in `LookupTablesStore.js` please?